### ISSUE TITLE
How can we SIMPLY load a ply mesh to CGAL, apply some operators and SIMPLY export another ply mesh? 

### ISSUE THREAD

**Issue by marwan-abdellah** [CGAL/cgal#3344]
_Please use the following template to help us solving your issue._

## Issue Details

How can we load a simple ply mesh, process it and the export the results back to ply mesh? It seems extremely complicated to do so? This is the first time that I have used CGAL, but that way and the poor documentation, I cannot figure it out myself.

**Comment by sgiraudot** [comment]
The IO functions for loading/writing PLY are not documented yet, but you can find them in [PLY_reader.h](https://github.com/CGAL/cgal/tree/master/Polyhedron_IO/include/CGAL/IO) and [PLY_writer.h](https://github.com/CGAL/cgal/tree/master/Polyhedron_IO/include/CGAL/IO) (the API should be simple enough).

Reading a mesh cannot be done directly as there is no guarantee that a PLY input will contain an orientable mesh (it might be a polygon soup), so you have to load a soup first and try to orient it afterwards (see [`orient_polygon_soup()`](https://doc.cgal.org/latest/Polygon_mesh_processing/group__PMP__orientation__grp.html#gaadc5bac3072b11d2313e40043c729d0a)).

You can find an example of how it's done [in the PLY plugin for Polyhedron demo](https://github.com/CGAL/cgal/blob/master/Polyhedron/demo/Polyhedron/Plugins/IO/PLY_io_plugin.cpp#L111).

**Comment by marwan-abdellah** [comment]
Thanks for your prompt reply. This answers my question.

**Comment by oleg-alexandrov** [comment]
With CGAL 4.14 (the latest at conda-forge), the  CGAL::write_PLY() function did not work for me. I did have a valid mesh, and the .off file was written successfully, but when writing a ply file the indices of the faces were the largest 64 bit integer, which was odd. 

The following worked though:  https://piero.dev/2017/04/how-to-write-a-cgal-polyhedron-to-ply-or-any-other-format/

which makes me think that write_PLY() is buggy. I did not test this with GDAL 5. 

It would be great if a standalone example is shown for how to write PLY files. It is a widely used format.
